packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -1147,7 +1143,7 @@ export async function createHotReloaderTurbopack(
       } else {
         // If the client is not connected, store the status so that we can send it
         // when the client connects.
-        cacheStatusesByRequestId.set(requestId, status)
+        cacheStatusesByRequestId.set(htmlRequestId, status)
```

Comment:
`cacheStatusesByRequestId` should be renamed then. Though I don't understand why we set clients by request ID but cache status by html request id.